pkg/pgclient/client.go

```diff
@@ -69,8 +68,7 @@ func NewClient(cfg *Config) (*Client, error) {
 		return nil, err
 	}
 
-	config := bigcache.DefaultConfig(10 * time.Minute)
-	metrics, _ := bigcache.NewBigCache(config)
+	metrics, _ := bigcache.NewBigCache(pgmodel.DefaultCacheConfig())
```

Comment:
kind-of unrelated to this pr, but do we want to have diff retention periods for series and metrics

Reply:
I would leave that decision for when we start doing some benchmarks and monitoring later on.